src/test/resources/assets/music_disc_test/sounds.json

```diff
@@ -0,0 +1,13 @@
+{
+  "test_sound_event": {
+    "sounds": [
+      {
+        "name": "music_disc_test:example_ogg",
+        "stream": true,
+        "__content": "First 10 seconds of 'The Ice Giants' by Kevin MacLeod",
+        "__link": "https://incompetech.filmmusic.io/song/5745-the-ice-giants",
+        "__license": "http://creativecommons.org/licenses/by/4.0/"
```

Comment:
Better to just reuse a vanilla sound so we don't have to deal with having other people's licenses and data in our repo.